src/nxdoc/nuxeo-server/additional-services/bulk-action-framework/bulk-actions-directory.md

```diff
@@ -64,3 +65,61 @@ curl -X POST -H 'Content-Type: application/json' -u Administrator:Administrator
         }'
 ```
 {{! /multiexcerpt}}
+
+### CSVExportAction
+
+It exports requested documents as a CSV File which can be sorted and ziped.
+
+System properties are always exported as first columns, then all the properties from provided schemas and xpaths are exported if they can be found.
+
+The following example shows how to use `csvExportAction` with java service:
+
+{{! multiexcerpt name='baf-csv-export-action-java-example'}}
+```
+// build command
+BulkCommand command = new BulkCommand.Builder(CSVExportAction.ACTION_NAME, "SELECT * from Document")
+                                       .repository("default")
+                                       .user("Administrator")
+                                       .param(CSVProjectionComputation.PARAM_SCHEMAS, ImmutableList.of("dublincore"))
+                                       .param(CSVProjectionComputation.PARAM_XPATHS, ImmutableList.of("cpx:complex/foo"))
+                                       .param(CSVProjectionComputation.PARAM_LANG, "fr") // default is context Locale
+                                       .param(SortBlob.SORT_PARAMETER, Boolean.FALSE) // default is TRUE
+                                       .param(ZipBlob.ZIP_PARAMETER, Boolean.TRUE) // default is FALSE
+                                       .build();
+
+
+
+// run command
+BulkService bulkService = Framework.getService(BulkService.class);
+String commandId = bulkService.submit(command);
+
+// await end of computation
+bulkService.await(commandId, Duration.ofMinutes(1));
+
+// get status
+BulkStatus status = bulkService.getStatus(commandId);
+```
+{{! /multiexcerpt}}
+
+You can submit the same command through REST and [Command endpoint]({{page page='command-endpoint'}}):
+
+{{! multiexcerpt name='baf-set-properties-action-operation-example'}}
+```
+curl -X POST -H 'Content-Type: application/json' -u Administrator:Administrator \
+    http://localhost:8080/nuxeo/api/v1/automation/Bulk.RunAction \
+    -d '{
+            "params":{
+                "query":"SELECT * FROM Document",
+                "action":"csvExport",
+                "parameters":{
+                    "schemas":[\"dublincore\"],
+                    "xpaths":[\"cpx:complex\/foo\"],
```

Comment:
Can't we have a useful example that works out of the box instead? like getting the blob filename and size?